deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17
+FROM golang:1.18
 ADD https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v1.4.3-static/docker-credential-gcr_linux_amd64-1.4.3.tar.gz /usr/local/bin/
```

Comment:
Would be worth updating these uses to v2.1.6, though docker-crendential-gcr don't seem to do -static releases now (last -static build from 2018!).  Maybe not required with modern Go builds which are static by default.

Reply:
done